cli/cmd/resources/namespace.go

```diff
@@ -23,23 +22,19 @@ func NewNamespace(name string) *v1.Namespace {
 }
 
 func GetOdigosNamespace(client *kube.Client, ctx context.Context) (string, error) {
-	namespaces, err := client.CoreV1().Namespaces().List(ctx, metav1.ListOptions{
+	configMaps, err := client.CoreV1().ConfigMaps("").List(ctx, metav1.ListOptions{
```

Comment:
For the migrartion period, let's keep them both.
first - try to search for namespace with label, and if not found, try with configmap.

If we make this change at once, new cli will have problem working with odigos version that has not yet migrated to the new lables.

Reply:
Cool, `odigos-deployment` is added in this PR, but `odigos-config` has the label and should work 👍